ocs_ci/ocs/acm/acm.py

```diff
@@ -414,15 +418,59 @@ def import_clusters_with_acm():
     kubeconfig_b = clusters_env.get("kubeconfig_location_c2")
     cluster_name_a = clusters_env.get("cluster_name_1")
     cluster_name_b = clusters_env.get("cluster_name_2")
+    clusters = ((cluster_name_a, kubeconfig_a), (cluster_name_b, kubeconfig_b))
     verify_running_acm()
-    driver = login_to_acm()
-    acm_nav = AcmAddClusters(driver)
-    acm_nav.import_cluster(
-        cluster_name=cluster_name_a,
-        kubeconfig_location=kubeconfig_a,
-    )
+    if import_ui:
+        driver = login_to_acm()
+        acm_nav = AcmAddClusters(driver)
+        acm_nav.import_cluster(
+            cluster_name=cluster_name_a,
+            kubeconfig_location=kubeconfig_a,
+        )
 
-    acm_nav.import_cluster(
-        cluster_name=cluster_name_b,
-        kubeconfig_location=kubeconfig_b,
-    )
+        acm_nav.import_cluster(
+            cluster_name=cluster_name_b,
+            kubeconfig_location=kubeconfig_b,
+        )
+    else:
+        for cluster in clusters:
+            log.info(f"Create project {cluster[0]}")
```

Comment:
@OdedViner , if we could print somewhere that "Importing clusters via CLI method" would be more readable